frontend/src/pages/cans/detail/BudgetSummary.jsx

```diff
@@ -48,18 +48,18 @@ const BudgetSummary = () => {
                                     <th>Amount</th>
                                 </tr>
                                 <tr>
-                                    <td>Total FY {cfy[0]?.fiscal_year || "--"} Funding</td>
-                                    <td>{cfy[0]?.total_fiscal_year_funding || "n/a"}</td>
+                                    <td>Total FY {can_fiscal_year[0]?.fiscal_year || "--"} Funding</td>
+                                    <td>{can_fiscal_year[0]?.total_fiscal_year_funding || "n/a"}</td>
                                 </tr>
                                 <tr>
                                     <td>Funded YTD</td>
-                                    <td>{cfy[0]?.amount_available || "n/a"}</td>
+                                    <td>{can_fiscal_year[0]?.amount_available || "n/a"}</td>
```

Comment:
I also noticed that `?` is always being used.  Why is that?  Is it because it's possible nothing could come back from the backend?

Reply:
Yeah, I went back and forth on whether to do the null handling here, or simply ensure the api endpoint would handle it and return an empty (fully structured) object.